sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -1157,7 +1157,7 @@
       "get": {
         "summary": "This method is used to retrieve feedback of a cloud-to-device message.",
         "description": "This method is used to retrieve feedback for cloud-to-device messages. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging for more information. This capability is only available in the standard tier IoT Hub. For more information, see [Choose the right IoT Hub tier](https://aka.ms/scaleyouriotsolution).",
-        "operationId": "HttpRuntime_ReceiveFeedbackNotification",
+        "operationId": "Messages_ReceiveFeedbackNotification",
```

Comment:
"Messages_RecieveFeedback" should be fine. "Notification" doesn't really add anything for me